inst/include/population_dynamics/fleet/fleet.hpp

```diff
@@ -93,14 +102,27 @@ namespace fims
       catch_at_age.resize(nyears * nages);
       catch_index.resize(nyears); // assume index is for all ages.
       age_composition.resize(nyears * nages);
+      log_Fmort.resize(nyears);
+      Fmort.resize(nyears);
+      log_q.resize(nyears);
+      q.resize(nyears);
     }
 
+    void Prepare(){
+      // for(size_t fleet_ = 0; fleet_ <= this->nfleets; fleet_++) {
+      // this -> Fmort[fleet_] = fims::exp(this -> log_Fmort[fleet_]);
+      for (size_t year = 0; year < this->nyears; year++)
+      {
+        this->Fmort[year] = fims::exp(this->log_Fmort[year]);
```

Comment:
Since surveys will be treated as fleets, will we be estimating Fmort for surveys?

Reply:
I'm honestly not sure about this. @cmlegault reminded me the issue with treating surveys as fleets is that then you need to account for surveys in allocation, so I'm not sure we will be able to get away with fully treating them the same. I'm inclined to leave this as a question/MQ issue though.